packages/playground-examples/copy/ko/3-8/Syntax and Messaging/Type Imports.tsx

```diff
@@ -0,0 +1,50 @@
+//// { compiler: { ts: "3.8.3" } }
+// 3.8에선 타입 import를 위해 새로운 구문들을 추가했습니다.
+// 이는 흐름에서 온 사용자와 유사합니다.
```

Comment:
```suggestion
// 3.8에선 타입 import를 위해 새로운 구문들을 추가했습니다.
// 이는 flow 사용자에게는 익숙할 것입니다.
```

여기서 flow 는 [flow - javascript 기반 타입언어](https://flow.org/) 를 의미하는 것 같습니다 :)

Reply:
아하! 그 flow를 뜻하는거였군요 ㅎㅎ 
피드백 고맙습니다. 수정해서 반영했습니다~ :)